packages/@lwc/template-compiler/src/parser/parser.ts

```diff
@@ -66,33 +90,22 @@ export default class ParserCtx {
         }
     }
 
-    withErrorWrapping<T>(fn: () => T, errorInfo: LWCErrorInfo, location: parse5.Location): T {
+    withErrorWrapping<T>(
```

Comment:
For a follow up PR. The `withErrorWrapping` is now only used in `parseExpression` expression to handle all the errors produced by the `invariant` calls. It would be preferable to replace the `invariant` calls with `ctx.throwAtLocation` to avoid having to rewrap the errors.

Reply:
I'll add this to the next PR.